README.md

```diff
@@ -54,53 +54,16 @@ Features include:
 - Sending of credentials on cross-origin requests (`withCredentials` flag - only has an effect on the client).
 - Request cancellation.
 
+## WebSocket
 
-## WHttp
-`WHttp` acts as an HTTP client that can be used to send many HTTP requests. Client-side this has no effect, but on the
-server this gives you the benefit of cached network connections.
+The WebSocket API mirrors the `dart:io.WebSocket` class to keep things simple. If you've used the server-side `WebSocket`
+class before, then you're ready to go. The benefit is that this same API works for client-side usage, as well, which is
+a big improvement over the `dart:html.WebSocket` class.
 
-Additionally, `WHttp` has static methods that make simple HTTP requests easy.
-
-```dart
-WHttp.get(Uri.parse('example.com'));
-WHttp.post(Uri.parse('example.com'), 'data');
-...
-```
-
-If you do create an instance of `WHttp`, make sure you close it when finished.
-
-```dart
-WHttp http = new WHttp();
-...
-http.close();
-```
-
-
-## WRequest
```

Comment:
I didn't feel like they added much that the documentation didn't cover. I was going to remove it in favor of adding a getting started guide that was more tailored towards new users instead of something for reference (https://github.com/Workiva/w_transport/issues/49)